store/src/java/com/zimbra/qa/unittest/TestMountpoint.java

```diff
@@ -22,66 +22,80 @@
 import java.util.List;
 import java.util.Set;
 
-import com.zimbra.common.util.StringUtil;
-import com.zimbra.cs.mailbox.Mailbox;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TestName;
+
 import com.zimbra.client.ZFolder;
 import com.zimbra.client.ZMailbox;
 import com.zimbra.client.ZMountpoint;
+import com.zimbra.common.util.StringUtil;
+import com.zimbra.cs.mailbox.Mailbox;
 
-import junit.framework.TestCase;
+public class TestMountpoint {
+
+    @Rule
+    public TestName testInfo = new TestName();
 
-public class TestMountpoint
-extends TestCase {
-    
     private static final String NAME_PREFIX = TestMountpoint.class.getName();
-    private static final String USER_NAME = "user1";
-    private static final String REMOTE_USER_NAME = "user2";
-    
-    public void setUp()
-    throws Exception {
-        cleanUp();
-    }
-    
-    public void tearDown()
-    throws Exception {
-        cleanUp();
+    private static String USER_NAME = "user1";
```

Comment:
These are duff initializations.  setUp() changes them to test specific values.  I can submit another pull request if you think it is important.  Certainly confusing.